packaging_automation/common_tool_methods.py

```diff
@@ -476,3 +480,26 @@ def remove_prefix(text, prefix):
         return text[len(prefix):]
     else:
         return text
+
+
+def initialize_env(exec_path: str, project_name: str, checkout_dir: str):
+    remove_cloned_code(f"{exec_path}/{checkout_dir}")
+    if not os.path.exists(checkout_dir):
+        run(f"git clone https://github.com/citusdata/{project_name}.git {checkout_dir}")
+
+
+def remove_cloned_code(full_checkout_dir: str):
+    if os.path.exists(f"{full_checkout_dir}"):
+        print(f"Deleting cloned code {full_checkout_dir} ...")
+        # https://stackoverflow.com/questions/51819472/git-cant-delete-local-branch-operation-not-permitted
+        # https://askubuntu.com/questions/1049142/cannot-delete-git-directory
+        # since git directory is readonly first we need to give write permission to delete git directory
+        run(f"chmod -R 777 {full_checkout_dir}/.git")
+        run(f"sudo rm -rf {full_checkout_dir}")
+        print("Done. Code deleted successfully.")
+
+
+def create_pr(gh_token: str, pr_branch: str, pr_title: str, repo_owner: str, project_name: str):
+    g = Github(gh_token)
+    repository = g.get_repo(f"{repo_owner}/{project_name}")
+    repository.create_pull(title=pr_title, base="master", head=pr_branch, body="")
```

Comment:
what about taking `base` as a parameter (defaulting to `master`) and using this function in other scripts too (like `prepare_release.py`) ?